indexer/drawing_rule_def.hpp

```diff
@@ -2,9 +2,21 @@
 
 #include "base/buffer_vector.hpp"
 
+#include "drape/utils/projection.hpp"
 
 namespace drule
 {
+
+// Priority range for area and line drules. Should be same as PRIORITY_RANGE in kothic.
```

Comment:
nit: Can we keep variable names identical for easier grepping?

Reply:
maybe next time I touch this code